apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-body.tsx

```diff
@@ -33,17 +22,16 @@ export const InAppBody = () => {
       render={({ field }) => (
         <FormItem className="w-full">
           <FormControl>
-            <InputField className="h-36 px-1">
-              <Editor
+            <InputField className="flex h-36 items-start">
+              <FieldEditor
```

Comment:
weird focus behaviour when clicking inside/outside of the field

https://github.com/user-attachments/assets/e7c722a8-8cb5-4d72-ad64-7c2cc8e8508b